MailResources/Localizable/fr.lproj/Localizable.strings

```diff
@@ -848,13 +848,16 @@
 "lockAppTitle" = "Application verrouillée";
 
 /* loco:6491b5d13be3801fbb056c22 */
-"lockedMailboxDescription" = "L’accès à votre adresse mail est actuellement bloqué. Nous vous invitons à contacter votre administrateur.";
+"lockedMailboxBottomSheetDescription" = "L’accès à votre adresse mail est actuellement bloqué.\nPour plus d’informations, consultez les FAQ.";
+
+/* loco:64999ba3e6d90f26b6023952 */
+"lockedMailboxBottomSheetTitle" = "L’adresse %@ est bloquée";
 
 /* loco:649adc1b599c6ce9d6099fa2 */
-"lockedMailboxDescription-plural" = "L’accès à vos adresses mail est actuellement bloqué. Nous vous invitons à contacter votre administrateur.";
+"lockedMailboxScreenDescription" = "Pour plus d’informations :";
```

Comment:
I have updated the name of both `lockedMailboxScreenTitle` and `lockedMailboxScreenDescription` because we never use the word "screen". Can you import once again loco and update the code where this string is used?

(Android is aware of this change)